components/outcomes/CategoricalReportBox.tsx

```
@@ -30,33 +32,56 @@ const CategoricalReportBox = ({
-      parseAssetId(assetIdString).unwrap() as CategoricalAssetId,
+    (assetIdString) => parseAssetIdStringWithCombinatorial(assetIdString)
+  );
+
+  // Filter to only categorical and combinatorial tokens, exclude scalar outcomes
+  const categoricalOutcomeAssets = outcomeAssets.filter(
+    (asset): asset is CategoricalAssetId | CombinatorialToken => 
+      !('ScalarOutcome' in asset)
   );
 
-  const [selectedOutcome, setSelectedOutcome] = useState(outcomeAssets[0]);
+  const [selectedOutcome, setSelectedOutcome] = useState<CategoricalAssetId | CombinatorialToken>(categoricalOutcomeAssets[0]);
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)